tests/jsdom/spec/XYPlot.spec.js

```diff
@@ -140,4 +140,23 @@ describe("XYPlot", () => {
       expect(Object.keys(chartProps[handler].args[0][0])).to.eql(expectedKeys);
     });
   });
+
+  it('renders SVG with defs when provided', () => {
+    const chart = mount(
+      <XYPlot
+        width={600}
+        height={800}
+        defs={
+          <linearGradient id="Gradient" x1="0" x2="0" y1="0" y2="1">
```

Comment:
I think this PR might be unnecessary. Can you just put this in as a child of XYPlot normally?